doc/source/Container.md

```diff
@@ -22,6 +22,18 @@ $ export CONTAINER_ENV_VARIABLE=CONVERSATION_SERVICE_WATSON_PIZZERIA
 $ export CLUSTER_NAME=<your_cluster_name>
 ```
 
+* Set the Kubernetes environment to work with your cluster:
+
+```
+bx cs cluster-config $CLUSTER_NAME
+```
+
+This output of this command will contain a KUBECONFIG environment variable that must be exported in order to set the context. Copy and paste this output in the terminal window. An example is:
```

Comment:
nit:  s/This output of this command/The output of this command/  I think the second s/this output/the output/ might also help a teeny bit so it doesn't look like "this" refers to the example instead of the output.